documentation/config_file_logbackxml.markdown

```diff
@@ -57,7 +57,7 @@ The user account that owns the Puppet Server process must have write permissions
 
 #### `scan` and `scanPeriod`
 
-Logback supports noticing and reloading configuration changes without requiring a restart, a feature Logback calls **scanning**. To enable this, set the `scan` and `scanPeriod` attributes in the `<configuration>` element of `logback.xml`.  Scanning can potentially degrade logging performance and is not enabled by default.
+Logback supports noticing and reloading configuration changes without requiring a restart, a feature Logback calls **scanning**. To enable this, set the `scan` and `scanPeriod` attributes in the `<configuration>` element of `logback.xml`:
```

Comment:
Can we mention that this is the default setting for logback.xml but does not apply to request logging?